pkg/rack/terraform.go

```diff
@@ -728,7 +728,11 @@ func terraformWriteTemplate(filename, version string, params map[string]interfac
 		}
 
 		output "provider" {
-			value = "{{.Provider}}"
+			value ="{{.Provider}}"
+		}
+
+		output "release" {
```

Comment:
`console` will use this when uninstalling a rack. currently, there's a bug because this output variable is not set so it will always try to uninstall using the latest changes in `master` which can have unexpected consequences.